.env.development

```diff
@@ -0,0 +1 @@
+BY_LANGUAGE=he
```

Comment:
provided as an example for dotenv config. To override it locally create a new file .env.development.local (should not be committed to git)